gui/control.js

```diff
@@ -26,10 +27,31 @@ function switchAccountHandler(nickname) {
   });
 }
 
+function checkAdminAccess(){
+    let exec = require('child_process').exec;
+    exec('NET SESSION', function(err,so,se) {
+        let warning_user = `
+            <div class="alert alert-parsec-danger" role="alert">
+              <h4 class="alert-heading">Need Admin privilege!</h4>
+              <p>Parsec switcher needs admin access to work :(</p>
+              Please launch program with admin rights. 
+              <hr>
+              <p class="mb-0">Want to report bug ? Report it here: https://github.com/DefineX-Studios/parsec-account-switcher</p>
+            </div>`
```

Comment:
better move html into index.html file and hide/show the dom here to not mix view and control